client/src/components/payment/payment-mixin.ts

```diff
@@ -240,25 +240,34 @@ export default class PaymentMixin extends Vue {
         const receipt = sbcPayment.receipts[0]
         await paymentModule.setPaymentReceipt(receipt)
       }
+      return true
     } catch (error) {
-      if (error instanceof PaymentApiError) {
-        await errorModule.setAppError({ id: 'payment-api-error', error: error.message } as ErrorI)
-      } else {
-        await errorModule.setAppError({ id: 'fetch-nr-payment-error', error: error.message } as ErrorI)
-      }
+      console.error('fetchNrPayment() =', error) // eslint-disable-line no-console
```

Comment:
I added this to most or all catch blocks. It displays an error in the console to help devs debug errors (other the actual error does not display anywhere).